http-common/src/server.rs

```diff
@@ -342,6 +342,23 @@ pub mod response {
         res
     }
 
+    pub fn chunked<S, O, E>(status_code: hyper::StatusCode, body: S) -> hyper::Response<hyper::Body>
```

Comment:
Perhaps either rename thing to `chunked_text` to make it clear it's specific to `text/plain`, or take the `&'static str` of the content type as a parameter. Either approach is fine; I'll sign off to not block.

Reply:
Ok, added the parameter.